docs/protodoc_manifest.yaml

```diff
@@ -49,9 +49,18 @@ fields:
     edge_config:
       note: >
         This timeout is not compatible with streaming requests.
-      example: 300s  # 5 mins
+      example: 300s # 5 mins
   envoy.extensions.filters.network.http_connection_manager.v3.HttpConnectionManager.use_remote_address:
     edge_config: {example: true}
+  envoy.extensions.filters.network.http_connection_manager.v3.HttpConnectionManager.normalize_path:
+    edge_config:
```

Comment:
FWIW I would probably open a tracking issue to reorganize this file. This was intended as a way to specify some example defaults, but I think you probably want a per-profile manifest (which looks a lot like this YAML) but is named after the profile.

Reply:
Related, what is the current documentation output of this file?

With the introduction of profiles, my documentation suggestion would be:
1) A page in the arch overview describing profiles and which ones are available as well as the default.
2) Some automation that takes each profile and spits out a page that shows the actual settings it effects (with ref links to the individual settings). This would allow cross referencing between the API tags that I think also refer to this file.